sdk/cosmos/azure-spring-data-cosmos/CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ### 3.22.0-beta.1 (Unreleased)
 
 #### Features Added
+* Added the `maxDegreeOfParallelism` flag to application.properties.
```

Comment:
Please see this entry in changelog for adding the PR reference as well to your changelog - https://github.com/Azure/azure-sdk-for-java/pull/28756/commits/74df7bbd382d90bc2d85ae569cb7fffc88a96f30#diff-c3d8155c7715e24fad4ab2a6849a9d128b261c1d7d95c981e3be58f4dda4f236R35
Also, would be good to explain a little bit more on this changelog - like, 

* Exposed `maxDegreeOfParallelism` feature from `CosmosQueryRequestOptions` through application.properties flag - See [PR 28756](https://github.com/Azure/azure-sdk-for-java/pull/28756)

Reply:
Done!